sklearn/linear_model/tests/test_omp.py

```diff
@@ -76,12 +76,16 @@ def test_unreachable_accuracy():
     assert_array_almost_equal(
         orthogonal_mp(X, y, tol=0),
         orthogonal_mp(X, y, n_nonzero_coefs=n_features))
-
-    assert_array_almost_equal(
-        assert_warns(RuntimeWarning, orthogonal_mp, X, y, tol=0,
-                     precompute=True),
-        orthogonal_mp(X, y, precompute=True,
-                      n_nonzero_coefs=n_features))
+    warning_message = (
+        "Orthogonal matching pursuit ended prematurely "
+        "due to linear\ndependence in the dictionary. "
```

Comment:
The `\n` is probably not intentional. Please feel free to fix the code that generate it to use a whitespace instead.

Reply:
Thanks for the review @ogrisel I have updated warning message in linear_model/_omp.py.

Reply:
Thanks!